packages/renderer/src/lib/ui/Select.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
hello, I was wondering if we should not call the component `Dropdown` as I think most of UI libraries are using that word (and it seems UX people are using that term as well)

Reply:
Also component colors need to be defined in the Podman Desktop design system made by  @ekidneyrh

Reply:
Sure, I can rename.

Reply:
Renamed. Added proposed component colors for the design system following the identical pattern to Input, and will review with @ekidneyrh and update if necessary before moving out of draft.